packages/dom/src/dom.js

```diff
@@ -93,7 +93,16 @@ function isEdge( container, isReverse, onlyVertical ) {
 		return false;
 	}
 
-	const rangeRect = getRectangleFromRange( selection.getRangeAt( 0 ) );
+	const range = selection.getRangeAt( 0 ).cloneRange();
+	const isForward = isSelectionForward( selection );
+	const isCollapsed = selection.isCollapsed;
```

Comment:
Nit: I'll leave it to your preference, but optionally destructure:

```js
const { isCollapsed } = selection;
```

Reply:
Sounds good!